website/source/api/metrics.html.md

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
What about this page: https://www.nomadproject.io/docs/agent/telemetry.html

Reply:
See https://github.com/hashicorp/nomad/pull/3162/files#diff-9351a387bcf0a0b62128bd332eb78cb4R15

Do you think the metrics endpoint should be documented on https://www.nomadproject.io/docs/agent/telemetry.html ?

Reply:
I meant the values in the table

Reply:
Ah, understood. 

See https://github.com/hashicorp/nomad/pull/3162/commits/e5410d106c17ea57da5130981261cc5b97202dd6 , let me know if there is anything I can add/change with this.